### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [Gediminas91/capstone#3]
It's best practice to add the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) on images used for a site considering the attribute is used both for [accessibility](https://www.codecademy.com/articles/accessibility) and as a backup, in case the image cannot be loaded. 

https://github.com/Gediminas91/capstone/blob/4226707eda8cac0f4eb1beab9877b9fb51c41ebf/index.html#L72
